sopel/modules/help.py

```diff
@@ -160,6 +160,13 @@ def setup(bot):
     bot.config.define_section('help', HelpSection)
 
 
+def shutdown(bot):
+    try:
+        del bot.memory['command-list']
+    except KeyError:
+        pass
```

Comment:
I'm not sure about deleting this one. The cache validation logic should take care of any settings changes or command additions/removals that would require regenerating the command list.